modules/lead/product-base/staging.tf

```diff
@@ -126,3 +130,75 @@ resource "kubernetes_role" "ci_staging_role" {
     verbs      = ["*"]
   }
 }
+
+
+resource "vault_database_secret_backend_connection" "mongodb" {
+  backend = "mongodb"
+  name    = "${var.product_name}-staging"
+
+  allowed_roles = [
+    local.vault_mongodb_role
+  ]
+
+  data = {
+    username = "root"
+    password = random_password.mongodb_root_password.result
+  }
+
+  mongodb {
+    connection_url = "mongodb://{{username}}:{{password}}@mongodb.${var.product_name}-db.svc.cluster.local/admin"
+  }
+
+  depends_on = [
+    helm_release.mongodb
+  ]
+}
+
+resource "vault_database_secret_backend_role" "mongodb_role" {
+  backend = "mongodb"
+  creation_statements = [
+    jsonencode({
+      db = "admin"
+      roles = [
+        {
+          role = "readWrite"
+          db   = "database"
```

Comment:
do we use a variable when configuring the database when standing up the mongodb instance?  if so, we should try to reuse that here.

Reply:
As far as I can tell we haven't actually started using these databases for market-summary so we aren't specifying which database we're using anywhere. I left it as `database` since we weren't using one yet and Mongodb [creates non-existent databases upon first use](https://docs.mongodb.com/manual/core/databases-and-collections/#create-a-database). 

If we want to go ahead and define a database here I can go about adding that configuration.

Reply:
Updated database value used in this secret backend role to match the database used in market-summary:
- [Connection string used by market-summary](https://github.com/liatrio/springtrader-marketsummary/blob/98edd158c2d155106be68001f72d0859db138d3d/src/repository/index.js#L16)
- [NODE_ENV value](https://github.com/liatrio/springtrader-marketsummary/blob/f5af8c0b8052a7c82814724cdf16f2baa430c644/Jenkinsfile#L27)